todo_app/data/trello_items.py

```diff
@@ -0,0 +1,120 @@
+import requests, dotenv, os
+
+from todo_app.Item import Item
+
+dotenv.load_dotenv()
+
+trello_key = os.getenv("API_KEY")
```

Comment:
Similar to the above, this works fine here but you may find it causes you headaches in M3 as these variables will likely get set _before_ your tests have time to set up fake environment variables. An easy solution to this is to wrap them in functions so we can delay the execution until the tests are actually ready to trigger things, e.g.:
```python
def trello_key():
    return os.getenv("API_KEY")
```
And then have the functions executed where you currently reference the variable:
```python
    query_params = {
        "key": trello_key(),
        "token": trello_api_token(),
        "fields": "name,id"
    }
```